packages/tools/fetch-tool/src/fluidAnalyzeMessages.ts

```diff
@@ -749,7 +756,7 @@ function calcChannelStats(
 function processQuorumMessages(
 	message: ISequencedDocumentMessage,
 	skipMessage: boolean,
-	sessionsInProgress: Map<string, ActiveSession>,
+	sessionsInProgress: Map<string | null, ActiveSession>,
```

Comment:
If the server joins the Quorum we're in trouble :-P. Can we guard at the callsites instead?